common/argo/base/vars.yaml

```diff
@@ -35,4 +35,15 @@ artifactRepository:
     # Key Format for objects stored by Workflows. This can reference Workflow variables
     keyFormat:
       required: true
-      default: artifacts/{{workflow.namespace}}/{{workflow.name}}/{{pod.name}}
\ No newline at end of file
+      default: artifacts/{{workflow.namespace}}/{{workflow.name}}/{{pod.name}}
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# Component: Argo configuration
+# Description: Various options to adjust Argo behavior
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+argo:
+  # Specifies the container runtime interface to use (default: docker)
+  # must be one of: docker, kubelet, k8sapi, pns
+  # Ref: https://github.com/argoproj/argo/blob/master/docs/workflow-controller-configmap.yaml
+  containerRuntimeExecutor:
```

Comment:
@aleksandrmelnikov this should just go under `application` as `containerRuntimeExecutor`.

Reply:
Actually, the more I think about it, having it under `workflowEngine` makes better sense. We may have other workflow specific fields in here.